src/main/java/com/exasol/releaserobot/usecases/validate/PlatformValidator.java

```diff
@@ -11,7 +11,6 @@
      * Validate a project.
      *
      * @param repository instance of {@link Repository}
-     *
      * @return validation report
      */
     public Report validate(final Repository repository);
```

Comment:
we can remove this interface as it does exactly the same as `RepositoryValidator`

Reply:
Removed it